packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -758,7 +774,7 @@ async function helmParameters(celoEnv: string, useExistingGenesis: boolean) {
     `--set celotool.image.repository=${fetchEnv('CELOTOOL_DOCKER_IMAGE_REPOSITORY')}`,
     `--set celotool.image.tag=${fetchEnv('CELOTOOL_DOCKER_IMAGE_TAG')}`,
     `--set domain.name=${fetchEnv('CLUSTER_DOMAIN_NAME')}`,
-    `--set genesis.genesisFileBase64=${Buffer.from(genesisContent).toString('base64')}`,
+    `--set genesis.useGenesisFileBase64="false"`,
```

Comment:
Are there any cases where this is true?

Reply:
Not deploying with celotool, but I preferred to keep the compatibility on the helm chart if someone wants to use directly with helm (and do not introduce a hard dependency with google storage).